apps/www/components/Account/NewsletterSubscriptions/index.js

```diff
@@ -47,7 +47,7 @@ export const NEWSLETTER_SETTINGS = gql`
 `
 
 const NewsletterSubscriptions = ({ t, isMember, free, onlyName }) => (
-  <Query query={NEWSLETTER_SETTINGS}>
+  <Query query={NEWSLETTER_SETTINGS} variables={{ newsletterName: onlyName }}>
```

Comment:
i know this is from before, but I find `onlyName` to be a confusing, erm, name. Maybe it could be called `newsletterName`everwhere?

Reply:
I think what supports the confusen is the switch to another wording once more (`newsletterName`) – I reworked it to keep `onlyName` from component prop down to query.